aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -211,35 +213,30 @@ impl VersionedValue {
         iter: &mut dyn DoubleEndedIterator<Item = (&TxnIndex, &CachePadded<AggregatorEntry>)>,
         delta: DeltaOp,
     ) -> Result<u128, MVAggregatorsError> {
+        use AggregatorApplyChange::*;
         use AggregatorEntry::*;
         use EstimatedEntry::*;
 
         let mut accumulator = delta;
         while let Some((idx, entry)) = iter.next_back() {
-            let delta = match &**entry {
-                Value(v, _) => {
+            let delta = match (&**entry, self.read_estimate_deltas) {
```

Comment:
I think having a match on self.read_estimate_deltas for bypass is cleaner?

Reply:
can you explain?